tcf_website/templates/base/base.html

```diff
@@ -56,6 +56,19 @@
     $("#wrapper").toggleClass("toggled");
   });
 
+  /*
+    Toggle sidebar on tap with mobile devices
+  */
+  if ($(window).width() <= 320) {
```

Comment:
This width is too narrow to support some mobile devices

Reply:
I attempted to find a screen width that would work for many devices. I'm not sure what the "perfect number" here is, could you offer some help? 480 perhaps?

Reply:
I'm too busy to properly investigate today, but I did some quick searching and found this in `static/base/base.css`: 
![image](https://user-images.githubusercontent.com/55100084/205516876-ea062bdd-3e6f-4990-8a4d-003130216a58.png)

So it seems like 576px might be it? I'm not really sure where these numbers came from though, or even if the units used in the code are actually px, so try looking into other parts to figure out how the sidebar is configured.